app/services/host_fact_importer.rb

```diff
@@ -31,24 +27,24 @@ def import_facts(facts, source_proxy = nil)
       facts_importer.import!
     end
 
-    skiping_orchestration do
+    skipping_orchestration do
       host.save(:validate => false)
     end
 
     parse_facts(facts, type, source_proxy)
   end
 
   def parse_facts(facts, type, source_proxy)
-    skiping_orchestration do
+    skipping_orchestration do
       host.parse_facts facts, type, source_proxy
     end
   end
 
   private
 
-  def skiping_orchestration(&block)
+  def skipping_orchestration(&block)
     if host.is_a?(Host::Managed)
-      host.without_orchestration_if(skip_orchestration?, &block)
+      host.without_orchestration(&block)
```

Comment:
no need to pass `&block` around, you can also drop it from the method signatures
```suggestion
      host.without_orchestration do
        yield
      end
```

Reply:
It seems to me bit more straight forward :thinking:
It is expecting a `block`, why shouldn't it have `block` in the signature?

Reply:
No strong feelings over it though

Reply:
Afaik capturing the block as a proc and storing it in a variable is slower than yielding into it directly, but I'm a bit foggy on the details why it is like this

Reply:
hmm didn't find anything on the topic, but I guess I'm ok with changing it :shrug: :)